debian/aeternity-node.links

```diff
@@ -0,0 +1,2 @@
+var/log/aeternity/node opt/aeternity/node/log
+opt/aeternity/node/docs usr/share/doc/aeternity-node/docs
```

Comment:
Work-arounds.  

Packaging tools will create the path `usr/share/doc/aeternity-node/` to populate changelog and similar files. We cannot link to it, because it exists. This requires the additional sub-directory.

The path is the standard place for documentation. From user's point of view it is a good idea to have a way to reach the docs from there.

We cannot copy the docs directly there without duplicating files on disk with the `docs`(Ubutnu/Debian context) file. The `install` file does not provide a way to exclude `docs/`. The files in the `_build` directory seem to vary and cause issues, so a wildcard is required instead of listing desired files.